dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/QuayImageRegistry.java

```diff
@@ -118,9 +125,27 @@ public QuayImageRegistry(final Token quayToken) {
                     throw new CustomWebApplicationException("Could not get QuayTag", HttpStatus.SC_INTERNAL_SERVER_ERROR);
                 }
             }
-            for (QuayTag tagItem : quayTags) {
+
+            // Search through the Quay tags for ones that are classified as a manifest list and then use its digest to get the repo's manifest.
+            List<QuayTag> cleanedQuayTagList = new ArrayList<>(quayTags);
+            Map<QuayTag, List<Image>> multiImageQuayTags = new HashMap<>();
+            Gson g = new Gson();
+            quayTags.stream().forEach(quayTag -> {
+                if (quayTag.isIsManifestList()) {
+                    try {
+                        List<Image> images = handleMultiArchQuayTags(tool, quayTag, g, cleanedQuayTagList);
+                        multiImageQuayTags.put(quayTag, images);
+                    } catch (ApiException ex) {
+                        System.out.println("d");
```

Comment:
Did you mean to just print "d"? Also, you should use a logger, and log the exception as well as whatever your message is.